schema/07-coingecko.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE token_values(
```

Comment:
I think it's better to rename this table `token_price` instead. This would probably make it easier to understand what it is used for